### DISCUSSION TITLE
Slurm-Web-Agent not compatible with debian 12

### DISCUSSION THREAD

**Discussion by Pols-HKV** [rackslab/Slurm-web#424] [Q&A] [answered]
I have installed slurm on debian 12 host and try to install slurm web, however the stable version of slurm for debian is 22.05 and slurm-web 4 is not compatible with it, how can i install an older version for slurm-web?
apt-get install slurm-web-agent=3.2.0 isn't working.

Thanks for the response.

**Comment by rezib** [comment, marked as answer]
Hello @Pols-HKV, I just converted the issue you opened #423 into the discussion as this is more a question than an actual issue.

Slurm-web v3 requires Slurm >= 23.02, Slurm-web v4 requires Slurm >= 23.11. This is explained in the documentation:

* For v4: https://docs.rackslab.io/slurm-web/overview/architecture.html#slurm-versions
* For v3: https://docs.rackslab.io/slurm-web/v3/overview/architecture.html#slurm-versions

Slurm-web is compatible with Debian 12 but not with the version of Slurm in Debian 12. You need to build your own packages of a more recent version of Slurm. The steps are given in official Slurm documentation: https://slurm.schedmd.com/quickstart_admin.html#debuild

By the way, it is still possible to install Slurm-web v3 on Debian but you need to add slurmweb-3 component in your sources and configure pinning priority, as explained in: https://docs.rackslab.io/slurm-web/v3/install/install/distribs/debian.html. This way, `apt` will automatically select the latest v3 release.

**Reply by Pols-HKV** [reply]
Dear Rezib, thank you for your response, your right its a discussion not an issue, sorry my mistake.

I did some searches but didn't find this page https://docs.rackslab.io/slurm-web/v3/install/install/distribs/debian.html, however i have still the error Server error: URL not found on slurmrestd: http+unix://slurmrestd///slurm/v0.0.40/ping in the webviewer, also if i set in agent.ini version=0.0.37.
so i have to find out how to build my own package, i will try to follow the manual, thank you.
